encoding.bs

```diff
@@ -2938,8 +2943,17 @@ and <var>code point</var>, runs these steps:
   <p class=note>If <var>pointer</var> is non-null, it is less than 8836 due to the nature of
   <a>index jis0208</a> and the <a>index pointer</a> operation.
 
- <li><p>If <var>pointer</var> is null, return <a>error</a> with
- <var>code point</var>.
+ <li>
+  <p>If <var>pointer</var> is null, then:
+
+  <ol>
+   <li><p>If <a>ISO-2022-JP encoder state</a> is <a lt="ISO-2022-JP encoder jis0208">jis0208</a>,
```

Comment:
Shouldn't this be

```suggestion
   <li><p>If <a>ISO-2022-JP encoder state</a> is not <a lt="ISO-2022-JP encoder ASCII">ASCII</a>,
```

Reply:
We don't do a state switch earlier when it's an error and Roman and I don't see why we should here. I should probably write some tests to ensure implementations do the right thing though.

Reply:
Please document (for non-browser users of the spec) that callers of the abstraction are responsible for emitting a replacement byte sequence that only uses the bytes that have the ASCII interpretation in the Roman state of ISO-2022-JP.